pandas/core/series.py

```diff
@@ -118,6 +118,8 @@ def wrapper(self):
 # ----------------------------------------------------------------------
 # Series class
 
+bool_t = bool  # Need alias because NDFrame has def bool:
```

Comment:
don't we already have this defined? @simonjayhawkins

Reply:
we have this in pandas.core.generic where bool method is defined. bool is not overridden here (or in core.frame) so don't need alias.

@charlesdong1991 u can remove this